pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -119,20 +138,30 @@ func createVerifyECDSASignatureHinter(resolver hintReferenceResolver) (hinter.Hi
 	if err != nil {
 		return nil, err
 	}
+
 	signature_r, err := resolver.GetResOperander("signature_r")
 	if err != nil {
 		return nil, err
 	}
+
 	signature_s, err := resolver.GetResOperander("signature_s")
 	if err != nil {
 		return nil, err
 	}
-	return newVerifyECDSASignatureHinter(ecdsaPtr, signature_r, signature_s), nil
+
+	return newVerifyECDSASignatureHint(ecdsaPtr, signature_r, signature_s), nil
 }
 
-func newGetPointFromXHinter(xCube, v hinter.ResOperander) hinter.Hinter {
+// GetPointFromX hint calculates a point on an elliptic curve given a x-coordinate
+//
+// `newGetPointFromXHint` takes 2 operanders as arguments
+//   - `xCube` is the x-coordinate value used to calculate the point
+//   - `v` is ?
```

Comment:
`v` represents the parity of `y` if I am not mistaken. It should be either 1 or 0

Reply:
oh great yes, makes sense! thanks!